src/pages/index.js

```diff
@@ -1,12 +1,29 @@
 import * as React from "react";
+import { Link } from "gatsby";
+
 import { DefaultLayout } from "../components/DefaultLayout";
+import { MyLink } from "../components/MyLink";
+import { Container } from "../components/Container";
 
 const IndexPage = () => {
   return (
-    <main className="container m-auto">
+    <Container>
       <title>Portfolio page</title>
-      <DefaultLayout></DefaultLayout>
-    </main>
+      <DefaultLayout>
+        <div className="grid lg:grid-cols-2 gap-x-32 my-56">
+          <div className="mb-32 lg:mb-0">left content</div>
+          <div>
+            <p className="text-3xl mb-5">Hi! Nice to meet you!</p>
+            <p className="text-violet-light mb-10">
+              My name is <span className="text-white">Nadiia</span> and I'm a
+              digital artist. Here I will have a short introductory text and
+              perhaps a few other things. Exciting!
+            </p>
+            <MyLink>Browse my work</MyLink>
```

Comment:
I recommend you use the default `Link`:

```jsx
import { Link } from 'gatsby'
```

Then, export the button styles from `./components/Button.jsx`:

```jsx
// components/Button.jsx

export const buttonStyles = 'bg-gradient-to-b from-violet to-violet-dark font-bold py-3 px-8 rounded-full w-max'

export function Button(props) {
  return <button {...props} className={buttonStyles} />
}
```

And then use `buttonStyles` on the `Link` directly on your index page:

```jsx
// src/pages/index.js
import { Link } from 'gatsby'
import { buttonStyles } from '../components/Button'

<Link to="#work" className={buttonStyles}>Browse my work</Link>
```

This way:

1. Your button styles are a variable which can be reused;
2. Your `pages/index.js` decides where the `Link` should lead `to`.

Reply:
I do not use Button so I will add properties directly  to the link